src/woeusb

```diff
@@ -2004,6 +2009,19 @@ util_check_function_parameters_quantity(){
 	return 0
 }; declare -fr util_check_function_parameters_quantity
 
+##Check every file in source directory for files bigger than max fat32 file size (~4GB)
+check_for_big_files(){
+	local source_fs_mountpoint="${1}"
+
+	for i in $(find ${source_fs_mountpoint}/ -type f); do
```

Comment:
* `"${quote}"` `"$(all)"` `"${the}"` `"$(expansions)"` (when possible)
* The trailing slash after `source_fs_mountpoint` is unnecessary
* The loop will fail when the source_fs contains filepaths with spaces, checkout [delimiter - Reading null delimited strings through a Bash loop - Stack Overflow](https://stackoverflow.com/questions/8677546/reading-null-delimited-strings-through-a-bash-loop) for a robust solution using process substitution and specifically configured `read` loop.
* For readability, avoid using `i` as a counter(?) variable when there are semantically better options like `file`